### PR TITLE
Bump version to 2.0.1 and update changelog

### PR DESCRIPTION
**Pull request by asaf-erlich**
# Description of change
Bump version to 2.0.1 and update changelog, changelog autogenerated using this command:
```
docker run -it --rm -v "$(pwd)":/usr/local/src/your-app docker.io/ferrarimarco/github-changelog-generator --user singer-io --project tap-jira --token $CHANGELOG_GITHUB_TOKEN --output CHANGELOG_ADDITION.md --since-tag v2.0.0 --future-release v2.0.1
```

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## [v2.0.1](https://github.com/singer-io/tap-jira/tree/v2.0.1) (2021-04-06)
+
+[Full Changelog](https://github.com/singer-io/tap-jira/compare/v2.0.0...v2.0.1)
+
+**Closed issues:**
+
+- Status names mapping to status subcategories [\#50](https://github.com/singer-io/tap-jira/issues/50)
+- primary key of Users stream needs to be updated [\#48](https://github.com/singer-io/tap-jira/issues/48)
```
Comment (dmosorast) [inline CHANGELOG.md:10]: I'd remove these closed issues, they were opened and closed by the submitters, not for code changes

**Review by dmosorast — CHANGES_REQUESTED** [review]
Just one change

File: CHANGELOG.md (hunk 1 above)
Comment (asaf-erlich) [reply CHANGELOG.md:10]: Sure

Comment (asaf-erlich) [reply CHANGELOG.md]: Done

**Review by dmosorast — APPROVED** [review]

**Pull request merged**